src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/CsvFormatResponseIT.java

```diff
@@ -163,4 +631,55 @@ private void setFlatOption(boolean flat) {
 
         this.flatOption = flat;
     }
+
+    private CSVResult executeCsvRequest(final String query, boolean flat) throws IOException {
+
+        return executeCsvRequest(query, flat, false, false, false);
+    }
+
+    private CSVResult executeCsvRequest(final String query, boolean flat, boolean includeScore,
+                                        boolean includeType, boolean includeId) throws IOException {
+
+        final String requestBody = super.makeRequest(query);
```

Comment:
Try to reuse `SQLIntegTestCase.executeQuery()`? I assume the only difference is extra params in request URL. We can add optional parameters in executeQuery for that. If not true, at lease reuse `SQLIntegTestCase.executeRequest()` for the last few lines.

Reply:
I started with using those two, then realized it's not possible without changing the interfaces of those, and since they are used in too many places, I didn't want to touch them. That would make this PR touch too many files.

Reply:
> I started with using those two, then realized it's not possible without changing the interfaces of those, and since they are used in too many places, I didn't want to touch them. That would make this PR touch too many files.

I don't think you need to "change" the interfaces. Basically you should override it to give the caller option to pass in URL parameters if they want.